libs/ledger-live-common/src/wallet-api/Exchange/server.ts

```diff
@@ -53,13 +60,13 @@ export type CompleteExchangeUiRequest = {
   amountExpectedTo?: number;
 };
 
-export type ExchangeStartParamsUiRequest = {
-  exchangeType: "FUND" | "SELL" | "SWAP" | "FUND_NG" | "SELL_NG" | "SWAP_NG";
+type ExchangeStartParamsUiRequest = {
+  exchangeType: "FUND" | "SELL" | "SWAP";
   provider?: string;
   exchange?: Exchange;
 };
```

Comment:
I'm not sure if it makes sense to change this type like that but it looks like we only want the provider and exchange param when `exchangeType !== "FUND"`
```suggestion
type ExchangeStartParamsUiRequest = {
  exchangeType: "FUND";
} & {
  exchangeType: "SELL" | "SWAP";
  provider: string;
  exchange: Exchange;
};
```